fhir-swagger-generator/src/main/java/com/ibm/fhir/openapi/generator/FHIROpenApiGenerator.java

```diff
@@ -1286,8 +1291,14 @@ private static void generateProperty(StructureDefinition structureDefinition, Cl
             JsonObjectBuilder wrapper = factory.createObjectBuilder();
             wrapper.add("type", "array");
             wrapper.add("items", property);
+            if (example != null) {
```

Comment:
So, looks like these following lines are the key of the fix. the old codes didn't really add the example. correct?

Reply:
not quite; the old code added the examples at the property level (lines 1276 and 1282 respectively).  now that we're adding the examples one level up, i actually meant to remove those lines...so thanks for the reminder!